src/mount/readdata.h

```diff
@@ -211,11 +210,19 @@ struct ReadRecord {
 	std::atomic<uint8_t> refreshCounter = 0;
 	std::atomic<uint16_t> requestsNotDone = 0;
 	bool expired = false; //gMutex
+	std::atomic<bool> stopThread;
+	std::thread garbageCollectorThread;
 
 	ReadRecord(uint32_t inode)
 	    : cache(gCacheExpirationTime_ms),
 	      readahead_adviser(gCacheExpirationTime_ms, gReadaheadMaxWindowSize),
-	      inode(inode) {}
+	      inode(inode),
+	      stopThread(false),
```

Comment:
If it is always initialized with false, please use member initializer instead of constructor parameter.

Reply:
Done